src/main/kotlin/pt/up/fe/ni/website/backend/config/upload/UploadConfigProperties.kt

```diff
@@ -0,0 +1,12 @@
+package pt.up.fe.ni.website.backend.config.upload
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "upload")
+class UploadConfigProperties(
+    val uploadType: String?,
```

Comment:
```suggestion
    val provider: String?,
```
upload seems redundant, do you think this is more explanatory?